index.html

```diff
@@ -0,0 +1,159 @@
+               <li class="box-1">DC Ignite</li>
+               <p class="box-2">Join our webinars and live training sessions.</p>
+            </div>
+          </div>
+        </ul>
+        <a class="mp-1">SEE THE WHOLE PROGRAM</a> 
+        <button class="button">Join Interior Symposium 2022</button>
+     </div>
+       
+    </section>
+
+      <section class=".section-speakers-heading container-fluid featured-speakers text-dark bg-light.bg-gradient">
+        <div class="row pt-3 mb-5">
+          <div class="col-12">
+            <h3 class="featured">Featured Speakers</h3>
+            <hr class="line hr-title">
+          </div>
+        </div>
+         
+      </section> 
+       <!-- <div class="col-12 text-center mb-5 border d-block d-md-none">
+            <a data-toggle="collapse" data-target="#demo" href="#" class="text-uppercase btn rounded-0 pr-5 pl-5 pt-2 pb-2">
+              More <i class="fas fa-angle-down red"></i>
+            </a>
+          </div> --> 
```

Comment:
Kindly remove commented code from your codebase to make for more readability of the code.